app/views/my/access_token_partials/_api_tokens_section.html.erb

```diff
@@ -0,0 +1,90 @@
+<%#-- copyright
+OpenProject is an open source project management software.
+Copyright (C) 2012-2023 the OpenProject GmbH
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See COPYRIGHT and LICENSE files for more details.
+
+++#%>
+<div class="attributes-group" id="api-token-section">
+  <div class="attributes-group--header">
+    <div class="attributes-group--header-container">
+      <h3 class="attributes-group--header-text"><%= t("my_account.access_tokens.api.title") %></h3>
+    </div>
+  </div>
+  <div>
+    <p>
+      <%= t("my_account.access_tokens.api.text_hint") %>
+    </p>
+    <% if Setting.rest_api_enabled? %>
+      <% if api_token %>
+        <div class="generic-table--container">
+          <div class="generic-table--results-container">
+            <table id="access-token-table" class="generic-table">
+              <%= render  partial: "my/access_token_partials/token_table_header",
+                          locals: { 
+                            column_headers: [
+                              t('attributes.name'), 
+                              User.human_attribute_name(:created_at), 
+                              t('my_account.access_tokens.headers.expiration')
+                            ] 
+                          } 
+              %>
+              <tbody>
+                <tr>
+                  <td style="width: 60%;"><%= t("my_account.access_tokens.api.static_token_name") %></td>
```

Comment:
The use of inline styles for static behaviour is discouraged in OpenProject. I do feel a bit bad about saying this as it might change with the inclusion of Primer soon (although I haven't found a simple table component in the DS but maybe I was just searching for the wrong string).

Not so much at this place but at the other tables (i.e. the iCalendar one) I would also prefer a more flexible approach as the Name, Calendar and Project table might vary in length depending on the contents.

Reply:
I decided to set the table column widths statically as we show multiple of these tables on the access token page and I wanted them to have the same optical structure. In my opinion it's easier to digest the information when the table columns appear on the same horizontal position:
![Screenshot from 2023-07-20 09-58-26](https://github.com/opf/openproject/assets/122434454/e9490f5f-00be-482b-aaac-ff08de849aa0)
If the table columns have dynamic widths, the horizontal position of the "created on" and "expires" column will be different for each table.

Reply:
I am ok with having static table widths for the use case you describe (multiple tables on the same pabe). But please do it using css classes as opposed to inline styles as e.d. done for the [subject column in the wp table where the `-max` modifier is applied](https://github.com/opf/openproject/blob/dev/frontend/src/global_styles/content/_table.sass#L137-L139). Some similar, maybe page specific css, can govern the access tokens page, which would also ensure that all of those keep the same value which is currently spread out between partials.